### ISSUE TITLE
Serenity emailable report is generated in wrong directory

### ISSUE THREAD

**Issue by scormaq** [serenity-bdd/serenity-core#1545]
If a project with serenity tests is a sub-project in Gradle multi-module project, serenity one-paged report (see here: [emailable reports](https://serenity-bdd.github.io/theserenitybook/latest/extended-reports.html)) is generated in root directory instead of sub-project.

**Serenity version: 2.0.34**
**System: Windows 10**
**Gradle: 5.1.1** 
**Java: 1.8.0_181**

**Steps to reproduce**
Here is demo project: https://github.com/scormaq/serenity-cucumber-gradle-multiproject
1. Clone it and execute following gradle command from root directory:
```bash
gradlew.bat :clean :smth:acceptance:clean :smth:acceptance:test :smth:acceptance:reports :smth:acceptance:aggregate
```
2. Verify where serenity reports generated

**Expected result**
Both regular and emailable reports are generated in sub-project (`${root}/smth/acceptance/target/site/serenity`)

**Actual result**
Regular serenity report is generated as expected (in `${root}/smth/acceptance/target/site/serenity`)

Emailable report is generated in root directory (in `${root}/target/site/serenity`)

However, in console gradle logs about that emailable and regular reports are generated both in sub-project:
```
    1 Scenarios (1 passed)
    1 Steps (1 passed)
    0m0.745s


> Task :smth:acceptance:reports
Generating Serenity Reports for acceptance to directory C:\My\Work\github\baseproj\smth\acceptance\target\site\serenity
PROCESSING EXTENDED REPORTS: [email]

> Task :smth:acceptance:aggregate
Generating Serenity Reports for acceptance to directory C:\My\Work\github\baseproj\smth\acceptance\target\site\serenity
```

**Comment by wakaleo** [comment]
Makes sense. Have a look at https://github.com/serenity-bdd/serenity-core/blob/d0af936b62ce1d4b235389a52c9339ab017162cd/serenity-gradle-plugin/src/main/groovy/net/serenitybdd/plugins/gradle/SerenityPlugin.groovy#L150 to see if you can find a fix.

**Comment by scormaq** [comment]
Hi @wakaleo, thanks for response.
I did some debugging around gradle plugin `serenity-emailer` and my latest conclusion is that this plugin does not get `EnvironmentVariables` correctly, i.e. next line does not do the job:
https://github.com/serenity-bdd/serenity-core/blob/d0af936b62ce1d4b235389a52c9339ab017162cd/serenity-emailer/src/main/kotlin/net/serenitybdd/reports/email/EmailReporter.kt#L28

I tried to configure property `serenity.outputDirectory` in all possible ways - in `test` task, in `serenity.conf` file, in `reports` task and so on. The line you pointed returns a correct result, so gradle logs a correct directory (as I mentioned in original post), however in the end output directory is set to `target/site/serenity` because of next:
https://github.com/serenity-bdd/serenity-core/blob/d0af936b62ce1d4b235389a52c9339ab017162cd/serenity-emailer/src/main/kotlin/net/serenitybdd/reports/email/SerenityEmailReport.kt#L18
https://github.com/serenity-bdd/serenity-core/blob/d0af936b62ce1d4b235389a52c9339ab017162cd/serenity-emailer/src/main/kotlin/net/serenitybdd/reports/email/SerenityEmailReport.kt#L25

The second line resolves output directory prior to `DEFAULT_OUTPUT_DIRECTORY` value because `environmentVariables` variable in `EmailReporter` resolves `serenity.outputDirectory` property to `null` despite of its configured value.
